status-network-contracts/src/rln/RLN.sol

```diff
@@ -117,6 +117,23 @@ contract RLN is Initializable, UUPSUpgradeable, AccessControlUpgradeable {
         }
     }
 
+    /// @dev Slashes identity with privateKey.
+    /// @param privateKey: RLN private key as bytes32;
+    /// @param rewardRecipient: Address that will receive the slash reward;
+    function _slash(bytes32 privateKey, address rewardRecipient) internal onlyRole(SLASHER_ROLE) {
+        // Hash the private key using Poseidon to get identityCommitment
+        uint256 identityCommitment = poseidonHasher.hash(uint256(privateKey));
+
+        User memory member = members[identityCommitment];
+        if (member.userAddress == address(0)) {
+            revert RLN__MemberNotFound();
+        }
+        karma.slash(member.userAddress, rewardRecipient);
+        delete members[identityCommitment];
+
+        emit MemberSlashed(member.index, msg.sender);
+    }
```

Comment:
This is a breaking change. Would you mind updating the commit accordingly with a breaking change section, as per  https://www.conventionalcommits.org/en/v1.0.0/#examples

An example can also be found here: https://github.com/status-im/status-network-monorepo/pull/85